server/internal/repository/eatery.go

```diff
@@ -13,6 +14,8 @@ type (
 		ID          uuid.UUID `db:"id"` // UUID
 		Name        string    `db:"name"`
 		Description *string   `db:"description"`
+		CreatedAt   time.Time `db:"createdAt"`
+		UpdatedAt   time.Time `db:"updatedAt"`
```

Comment:
データベース内では`created_at`のような名前になっているのでそこだけ修正お願いします :pray:
```suggestion
		CreatedAt   time.Time `db:"created_at"`
		UpdatedAt   time.Time `db:"updated_at"`
```